### PR TITLE
Initial document list fed

### PR DESCRIPTION
**Pull request by Blunderchips**
- Install material-table
- Install UUID
- Format index.html
- Import material icon set on index.html
- Add docs route, link to DocTable.tsx
- Remove ncu script, not needed

### CODE REVIEW

**Comment by Blunderchips** [comment]
- https://material-ui.com/components/tables/#material-table
- https://www.w3.org/WAI/tutorials/tables/
- https://github.com/mui-org/material-ui/blob/master/docs/src/pages/components/tables/MaterialTableDemo.js
- https://www.npmjs.com/package/material-table
- https://www.npmjs.com/package/uuid
- https://github.com/mbrn/material-table/issues/547

File: package.json [hunk 1]
```diff
@@ -46,9 +46,11 @@
     "@types/react-swipeable-views": "0.13.0",
     "@types/testing-library__react": "9.1.2",
     "@types/testing-library__user-event": "4.1.1",
+    "@types/uuid": "3.4.6",
```
Comment (ToeFungi) [inline package.json]: Move to `devDependencies` as this is not a dependency once it's compiled to JS

File: src/modules/doc/DocTable.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+/**
+ * Table of all the user's documents.
+ *
+ * @since 0.4.12
+ *
+ * @see https://material-ui.com/components/tables/#material-table
+ */
+// https://github.com/mui-org/material-ui/blob/master/docs/src/pages/components/tables/MaterialTableDemo.js
+const DocTable: React.FC<any> = (props) => {
```
Comment (ToeFungi) [inline src/modules/doc/DocTable.tsx]: Can you not add a typing instead of `any`?

File: src/modules/doc/DocTable.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+    const {
+        push,
+    } = props;
+
+    const [state, setState] = React.useState<{ columns: Array<Column<any>>, data: Array<any> }>({
```
Comment (ToeFungi) [inline src/modules/doc/DocTable.tsx]: Can you not add a typing instead of `any`?

**Comment by Blunderchips** [comment]
why is merging so hard

**Pull request merged**